### ISSUE TITLE
multi line output does not match README

### ISSUE THREAD

**Issue by mapleoin** [timothycrosley/isort#279]
Trying to change the multiline output does not seem to do anything. I tried with real-world examples. But even the README example seems to fail:

```
$ isort -m3 -d example.py                         
from thirdparty import lib1, lib2, lib3, lib4, lib5, lib6
$ isort -m2 -d example.py
from thirdparty import lib1, lib2, lib3, lib4, lib5, lib6
$ isort -m1 -d example.py
from thirdparty import lib1, lib2, lib3, lib4, lib5, lib6
$ cat example.py
from thirdparty import lib1, lib2, lib3, lib4, lib5, lib6
$ isort -v
isort 3.9.6
 ```

**Comment by patrickyevsukov** [comment]
From the README.md file: " ... wraps long from imports to the specified line length (defaults to 80)."

You are not passing a line length parameter, and the line is less than 80 chars. It seems isort is behaving as expected. 

You might try passing a line length argument:

```shell

patrick@ubuntu ~$ cat test.py
 
from thirdparty import lib1, lib2, lib3, lib4, lib5, lib6

patrick@ubuntu ~$ isort -m 3 -d -w 40 test.py

from thirdparty import (
    lib1,
    lib2,
    lib3,
    lib4,
    lib5,
    lib6
)

```

For a full list of options, have a peek at the isort source code here (beginning at line 102): https://github.com/timothycrosley/isort/blob/develop/isort/main.py

**Comment by mapleoin** [comment]
Ah, interesting. I didn't think of the line length. So I have a follow-up question: Is there any way to force splitting on multiple lines (*Vertical Hanging indent*) without also forcing this:

```python
from thirdparty import \
    just_one_module
```

That's what happened when I tried `isort -m3 -w1`. I was trying to force all multiple submodule imports to be on their own line.

Also, I still think the README is misleading. It should probably use a better example which breaches the (I assume) default 80chars limit.

**Comment by patrickyevsukov** [comment]
Indeed there is; but, not with this version of isort. Take a look at issue #267. I have merged this functionality into the Truveris fork of isort.

In a new venv, run:
```shell
pip install git+ssh://git@github.com/truveris/isort.git
```

This code:
```python
from thirdparty import \
    just_one_module
```

Will be formatted like this, using the Truveris fork:
```python
from third party import (
    just_one_module,
)
```

**Comment by mapleoin** [comment]
Cool. thank you @patrickyevsukov , that looks like the right behaviour. I would like to leave this issue open until that functionality is merged into this project and both your second example and my first example work in the same project. 

I would actually prefer having an option which is indents the style of Vertical Hanging Indent regardless of the line length.

**Comment by timothycrosley** [comment]
This is fixed in version 4.0 of isort just released. Thanks!

~Timothy

**Comment by mapleoin** [comment]
Cool. I got some time to look at the way this works in the latest version. So just to answer my initial question about how to get the output from the README (in case anyone else wanders here later):

```
$ cat isort.py 
from thirdparty import lib1, lib2, lib3, lib4, lib5, lib6
$ isort --stdout --force-grid-wrap --multi_line 3 --trailing-comma isort.py 
from thirdparty import (
    lib1,
    lib2,
    lib3,
    lib4,
    lib5,
    lib6,
)
```

I still think the README needs some work, as it makes it sound like only the `--multi-line 3` option is required for the ouput above. But I'm happy that I can now use isort for my usecase.
